tests/sentry/sentry_metrics/querying/data/test_api.py

```diff
@@ -1432,13 +1434,15 @@ def test_span_metrics_with_coercible_units(self):
                 ],
             },
         ]
+        objects = []
         for config in configs:
-            self.create_span_attribute_extraction_config(
+            o = self.create_span_attribute_extraction_config(
                 dictionary=config, user_id=self.user.id, project=self.project
             )
+            objects.append(o)
 
-        mri_1 = "d:custom/span_attribute_1@nanosecond"
-        mri_2 = "d:custom/span_attribute_2@microsecond"
+        mri_1 = f"d:custom/span_attribute_{objects[0].conditions.all().get().id}@none"
+        mri_2 = f"d:custom/span_attribute_{objects[1].conditions.all().get().id}@none"
```

Comment:
Nit: you don't need to use `all()` before using `get()`, just `get()` is enough
```suggestion
        mri_1 = f"d:custom/span_attribute_{objects[0].conditions.get().id}@none"
        mri_2 = f"d:custom/span_attribute_{objects[1].conditions.get().id}@none"
```